Source/SvgOptions.cs

```diff
@@ -1,31 +1,177 @@
-﻿using System.Collections.Generic;
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+#if NETCORE
+using System.Diagnostics.CodeAnalysis;
+#endif
 
-#nullable enable
-
-namespace Svg;
-
-public class SvgOptions
+namespace Svg
 {
-    public SvgOptions()
+    public class SvgOptions : IDictionary<string, string>, ICloneable
```

Comment:
I think it would be better to inherit `Dictionary`.

Reply:
Just wanted it to be as dictionary, we might change the base (and internal) to anything later.